### PR TITLE
Repair tab complete and stats gui and required lobbyloc bedore join to arena

### PR DESCRIPTION
**Pull request by wiktorekx**

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java [hunk 1]
```diff
@@ -419,6 +419,9 @@ public boolean addPlayer(Player p, boolean skipOwnerCheck) {
         /* used for base enter/leave event */
         isOnABase.remove(p);
         //
+        if(getServerType() != ServerType.BUNGEE && config.getConfigLoc("lobbyLoc") == null){
```
Comment (Memexurer) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java:422]: what does this code change?

Comment (wiktorekx) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java:422]: Checks if lobbyLoc is set

Comment (wiktorekx) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java:422]: Checks if lobbyLoc is set

Comment (Memexurer) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java:422]: y tho

Comment (wiktorekx) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java:422]: Protection against error on player leave

Comment (Memexurer) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java:422]: but this code is getting executed on arena join

Comment (Memexurer) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java:422]: also why dont you just set lobby location on your bw setup

**Pull request closed without merging**